src/mappings/entities/common.ts

```diff
@@ -14,3 +14,13 @@ export interface HandlerArgs {
   params: Codec[];
   event: SubstrateEvent;
 }
+
+export const getAsset = async (ticker: string): Promise<Asset> => {
+  const asset = await Asset.getByTicker(ticker);
+
+  if (!asset) {
```

Comment:
IMO this should have been in `mapAsset` itself. I would prefer we input entity types in their respective mappers only (until unless we can't).

Reply:
I moved it from `mapAsset`. I think its better to have the functions that are imported into other handlers in a clear common folder, but I agree the shared functions should be minimized.

I think enough things link to asset for this to be an exception. Its pretty small, but doesn't seem worth it to duplicate in each handler either.